src/components/Register/Register.js

```diff
@@ -26,7 +27,7 @@ if (showSuccess) {
 
 if (!showSuccess) {
     return (
-        <> 
+        <PopupWithForm closeModal={closeModal} isModalOpen={isModalOpen} setShowSuccess={setShowSuccess}> 
         <h3 className="popup__title">Sign up</h3>
```

Comment:
You need to format the markup to show indentation and to make the code more readable. You can do that with `prettier` or a combination `SHIFT + ALT + F`

```jsx


    return (
        <PopupWithForm closeModal={closeModal} isModalOpen={isModalOpen} setShowSuccess={setShowSuccess}> 
            <h3 className="popup__title">Sign up</h3>
            <form onSubmit={handleRegistration} className="popup__form">

```